Makefile

```diff
@@ -100,9 +119,8 @@ cluster-sync-resources: $(KUBECTL)
 		$(KUBECTL) apply -f deploy/openshift/; \
 	fi
 
-cluster-sync-handler: cluster-sync-resources $(local_handler_manifest)
-	IMAGE_REGISTRY=localhost:$(shell $(CLI) ports registry | tr -d '\r') \
-		make push-handler
+cluster-sync-handler: IMAGE_REGISTRY = localhost:$(shell $(CLI) ports registry | tr -d '\r')
```

Comment:
Please move unrelated changes out. It is hard to review this PR for me since there are so many changes I'm not so confident reviewing.

Reply:
Done